specs/error_state/ErrorOutOfGasSHA3.md

```diff
@@ -0,0 +1,45 @@
+# ErrorOutOfGasSHA3 state
+
+## Procedure
+
+Handle the corresponding out of gas errors for `SHA3`.
+
+### EVM behavior
+
+The `SHA3` gas cost is calculated as:
+
+```
+minimum_word_size = (size + 31) // 32
+
+static_gas = 30
+dynamic_gas = 6 * minimum_word_size + memory_expansion_cost
+```
```

Comment:
Maybe we could explicitly mention that `gas_cost = static_gas + dynamic_gas`, otherwise `static_gas` looks unused.

Reply:
fixed in [4952aad](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/477/commits/4952aad1ef387af524274b61f23c3b1246aeaf60)